### ISSUE TITLE
`composer show` treats the working directory as the install path for packages without a specific path, such as metapackages

### ISSUE THREAD

**Issue by phenaproxima** [composer/composer#11389]
Some package types, namely metapackages, don't have a particular install path. If you have any such packages installed, and you run `composer show --format=json --path`, you'll see that, for those metapackages, Composer will report the root directory of the project as the metapackage's install path.

For example, try the following (`drupal/core-recommended` is a metapackage):

```
composer create-project drupal/recommended-project
cd recommended-project
composer show --path drupal/core-recommended
```

The path will be reported as `/PATH/TO/recommended-project`.

The question: is this behavior intentional? Would it make more sense for this to be `null`, since the package does not truly have a path? Or is it supposed to be null, but this is a bug?

Any clarification will be hugely appreciated! Thanks in advance :)

**Comment by Seldaek** [comment]
Yeah I'm guessing maybe this can be fixed by using an explicit null somewhere, but maybe that'll break things too, gotta try it and investigate a bit more.
